openfisca_france/model/prestations/aides_logement.py

```diff
@@ -274,48 +274,111 @@ def function(self, simulation, period):
 
         return period, ressources
 
-class aide_logement_loyer_retenu(Variable):
+class aide_logement_loyer_plafond(Variable):
```

Comment:
Il faut alors faire en sorte de bien spécifier la plage temporelle des variables intermédiaires. En effet l'intérêt des helpers est que l'on peut facilement voir quand ils interviennent (dans le temps) alors que l'on se retrouve souvent avec des variables dont on ne trace pas facilement l'origine ni le fait qu'elles ne sont qu'un calcul intermédiaire. Bref la lisibilité is in the eye of the beholder ;-)